### DISCUSSION TITLE
Is there a way to add rounded corners to the dropdown menu?

### DISCUSSION THREAD

**Discussion by yasinalibd02** [rabbihossen/dropdown_pro#5] [Q&A] [answered]
I want the dropdown to look more modern with rounded corners. Any suggestions on how to do this?

**Comment by rabbihossen** [comment, marked as answer]
To add rounded corners, use the menuRadius property in DropdownPro. Setting menuRadius will apply rounded corners to the dropdown menu, making it look more modern:

```dart
DropdownPro<String>(
  itemsList: items,
  enableSearch: true,
  onChanged: (item) {
    setState(() {
      selectedItem = item;
    });
  },
  menuRadius: 10,  // Sets rounded corners with a radius of 10
)

```
